entries/b/blackbaud.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Blackbaud": {
+    "domain": "blackbaud.com",
+    "img": "blackbaud.com.png",
+    "tfa": [
+      "email",
```

Comment:
Where is email MFA listed? I don't see it in their docs.

Reply:
Ah! I misread a paragraph on their page 😅

```suggestion
```